src/components/atoms/Toast/Toast.tsx

```diff
@@ -36,22 +36,20 @@ const Toast = ({ className, header, info, type = 'error' }: IToast): JSX.Element
             <ToastIconInfo />
           </ToastIconWrapper>
           <div>
-            <ToastHeader>{!header ? DefaultToastHeaders.INFO : header}</ToastHeader>
-            <ToastInfo>{!info ? DefaultToastInfo.INFO : info}</ToastInfo>
+            <ToastHeader>{header || DefaultToastHeaders.INFO}</ToastHeader>
+            <ToastInfo>{info || DefaultToastInfo.INFO}</ToastInfo>
           </div>
         </ToastContent>
       );
     default:
       return (
         <ToastContent data-testid="toastDefault" className={className}>
+          <ToastIconWrapper>
+            <ToastIconError />
+          </ToastIconWrapper>
           <div>
-            <ToastIconWrapper>
-              <ToastIconError />
-            </ToastIconWrapper>
-            <div>
-              <ToastHeader>{!header ? DefaultToastHeaders.ERROR : header}</ToastHeader>
-              <ToastInfo>{!info ? null : info}</ToastInfo>
-            </div>
+            <ToastHeader>{header || DefaultToastHeaders.ERROR}</ToastHeader>
+            <ToastInfo>{info || null}</ToastInfo>
```

Comment:
```suggestion
            {info && <ToastInfo>{info}</ToastInfo>}
```

Reply:
Make sense but why this way is better?

Done